interlok-azure-onedrive/src/main/java/com/adaptris/interlok/azure/onedrive/OneDriveConsumer.java

```diff
@@ -94,7 +95,9 @@ protected int processMessages()
       {
         AdaptrisMessage adaptrisMessage = getMessageFactory().newMessage();
         InputStream remoteStream = graphClient.users(username).drives(oneDrive.id).items(driveItem.id).content().buildRequest().get();
-        IOUtils.copy(remoteStream, adaptrisMessage.getOutputStream());
+        OutputStream outputStream = adaptrisMessage.getOutputStream();
```

Comment:
try-with-resources here, otherwise you may never close the streams...

Reply:
Done.